.github/workflows/lint.yaml

```diff
@@ -0,0 +1,48 @@
+name: Linting
+
+on: [push, pull_request]
+
+jobs:
+
+  rubocop:
+    name: Rubocop
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+      - name: Setup Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.5.x'
+      - name: Install rubocop
+        run: |
+          gem install rubocop
+          which rubocop
```

Comment:
Do we need the `which rubocop`? More just curious, we don't need to remove it.

Reply:
Whoops, nope. That was from some testing to see if I could cache rubocop. I'll remove it.